charts/nginx/templates/nginx-configmap.yaml

```diff
@@ -48,14 +48,16 @@ data:
 
         -- Split the query string into individual parameters
         for param in string.gmatch(queryString, "([^&]+)") do
-          local paramName, paramValue = string.match(param, "(.+)=(.+)")
+          local paramName, paramValue = string.match(param, "([^=]+)=(.*)") -- Allow empty paramValue
 
           -- Check if the parameter should be masked
           if paramsToMask[paramName] then
-              paramValue = "REDACTED" -- Replace the value with REDACTED
+              paramValue = "##REDACTED##" -- Replace the value with REDACTED
```

Comment:
Same as below
```suggestion
              paramValue = "**REDACTED**" -- Replace the value with REDACTED
```

Reply:
yes this is fine i added ## since in astro-ui we were using same convention

Reply:
ok, let's keep it consistent for now. we can change this in a future release.